glotaran/builtin/models/kinetic_spectrum/kinetic_spectrum_result.py

```diff
@@ -53,6 +54,13 @@ def finalize_kinetic_spectrum_result(
         retrieve_decay_assocatiated_data(model, dataset, dataset_descriptor, "spectra")
 
         irf = dataset_descriptor.irf
+        if isinstance(irf, IrfMultiGaussian):
+            if isinstance(irf.center, list):
+                dataset['irf_center'] = irf.center[0].value
```

Comment:
Why not store the full list?

Reply:
In the word of Raymond Hettinger, "There must be a better way". :)

As discussed, a better way is to address the core issue, which is the missing parameters from the result object. And further, the issue of not being able to reproduce the analysis from the output of the saved nc file.